package.json

```diff
@@ -54,11 +54,9 @@
     "package.json": "npx sort-package-json"
   },
   "dependencies": {
-    "@metamask/eth-sig-util": "^5.0.0",
     "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#feature/PP-278/PP-329/upgrade_contracts",
-    "axios": "~1.1.2",
+    "axios": "~0.27.2",
```

Comment:
Should we add a task in the backlog to update the axios version?

Reply:
yes I think so, I tried but it was time consuming so I decided to follow the easy path.